### PR TITLE
Fixes #15116: Document use G1GC to better deal with high volume of nodes and reports

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/15116

### CODE REVIEW

File: src/reference/modules/administration/pages/performance.adoc [hunk 1]
```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
 The amount of memory should be the half of the RAM of the server, rounded up to the nearest GB.
 For example, if the server has 5GB of RAM, 3GB should be allocated to Jetty.
 
+When RAM allocated to jetty reach 4 to 6GB (or higher), you may experience long freeze of Rudder, up 
+to several tens of seconds. In this is the case, you can change the JVM garbage collector 
+used to one better fitted for the kind of memory size:
```
Comment (ncharles) [inline src/reference/modules/administration/pages/performance.adoc]: "for larger memory footprint" ?

File: src/reference/modules/administration/pages/performance.adoc [hunk 2]
```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
...
+----
+
+# edit +/etc/default/rudder-jetty+ with your preferred text editor, for example vim:
+vim /etc/default/rudder-jetty
+
+Notice: that file is alike to +/opt/rudder/etc/rudder-jetty.conf+, which is the file with
+default values. +/opt/rudder/etc/rudder-jetty.conf+ should never be modified directly because
+modification would be erased by packaging in the following Rudder versuib update.
+
+# Uncomment the lines related to "Java Garbage collector optiont"
```
Comment (ncharles) [inline src/reference/modules/administration/pages/performance.adoc]: typo: option

File: src/reference/modules/administration/pages/performance.adoc [hunk 3]
```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
...
+used to one better fitted for the kind of memory size:
+
+----
+
+# edit +/etc/default/rudder-jetty+ with your preferred text editor, for example vim:
+vim /etc/default/rudder-jetty
+
+Notice: that file is alike to +/opt/rudder/etc/rudder-jetty.conf+, which is the file with
+default values. +/opt/rudder/etc/rudder-jetty.conf+ should never be modified directly because
+modification would be erased by packaging in the following Rudder versuib update.
```
Comment (ncharles) [inline src/reference/modules/administration/pages/performance.adoc]: in this file, in existing install, there is 
-XX:+CMSClassUnloadingEnabled
-XX:+UseConcMarkSweepGC


which one is taken into account then if we add the option below ?

Comment (fanf) [reply src/reference/modules/administration/pages/performance.adoc]: The parent ticket modified that file to use by default (ie if `JAVA_GC` is not defined) `CMSGC`, and to use whatever the user defined for it in the other case. 
See https://github.com/Normation/rudder-packages/pull/1971/

**Comment by fanf** [comment]
PR updated with a new commit

File: src/reference/modules/administration/pages/performance.adoc [hunk 4]
```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
 The amount of memory should be the half of the RAM of the server, rounded up to the nearest GB.
 For example, if the server has 5GB of RAM, 3GB should be allocated to Jetty.
 
+When RAM allocated to jetty reach 4 to 6GB (or higher), you may experience long freeze of Rudder, up
```
Comment (amousset) [inline src/reference/modules/administration/pages/performance.adoc]: reaches

Comment (amousset) [inline src/reference/modules/administration/pages/performance.adoc]: long freezes

File: src/reference/modules/administration/pages/performance.adoc [hunk 5]
```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
 The amount of memory should be the half of the RAM of the server, rounded up to the nearest GB.
 For example, if the server has 5GB of RAM, 3GB should be allocated to Jetty.
 
+When RAM allocated to jetty reach 4 to 6GB (or higher), you may experience long freeze of Rudder, up 
+to several tens of seconds. In this is the case, you can change the JVM garbage collector
```
Comment (amousset) [inline src/reference/modules/administration/pages/performance.adoc]: If this is the case

File: src/reference/modules/administration/pages/performance.adoc (hunk 3 above)
Comment (ncharles) [reply src/reference/modules/administration/pages/performance.adoc]: you need to state that this solution is for 5.0.12 and later only then

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**